x/wasm/internal/keeper/wasmtesting/mock_engine.go

```diff
@@ -138,6 +140,21 @@ func (m *MockWasmer) Cleanup() {
 	m.CleanupFn()
 }
 
+func (m *MockWasmer) Pin(checksum wasmvm.Checksum) error {
```

Comment:
Let's just make these noops? Or counters?
I guess this is consistent, but feel it is easier to do nothing here

Reply:
No test to touch for this (yet)